### ISSUE TITLE
[Bug] <Spotube doesn't start on Linux mint Tricia>

### ISSUE THREAD

**Issue by EmaG17** [KRTirtho/spotube#349]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When i download and install Spotube (appimage or .deb package) nothing happens, even when installation goes like it should. 

### Expected Behavior

For the pictures in the program description, i expect that just works. 

### Spotube Version

At least in the last two versions. 

### Operating System

Linux

### Linux Distribution (if applicable)

Mint

### Operating System Version

18

### Installation Source

Official Website (spotube.netlify.app), Github Release (Binary)

**Comment by etckra** [comment]
Similar issue, Arch Linux, when using the system title bar it will no longer close but when using the app title bar, just launches an empty window. Came with the newest update and was working just fine beforehand. Uninstalled and reinstalled several times via yay pamac and flatpak. Just doesnt fucking work anymore.

**Comment by arturopitu** [comment]
Same issue as @etckra, the second time you open the app only a blank window appear, and for me killing the process and relaunching seems to make it work again. Debian 12 with the deb and flatpak version.

**Comment by Mikesco3** [comment]
I'm running Linux Manjaro on Xfce4 and the flatpak is on version 3.3 
However when I Installed the version 3.4 from the AUR then I signed into lastfm, everything was fine, but when I signed into spotify with the cookie, it went gray... 
![image](https://github.com/KRTirtho/spotube/assets/13419518/f60ab0ea-d078-462f-8e1d-237e96fa4112)
___
I tried starting it from the command prompt and I get this:
```
(process:239219): GLib-GObject-CRITICAL **: 20:49:29.314: g_object_new_is_valid_property: object class 'MyApplication' has no property named 'com.github.KRTirtho.Spotube'

(spotube:239219): Gdk-CRITICAL **: 20:49:29.374: gdk_window_get_state: assertion 'GDK_IS_WINDOW (window)' failed
package:media_kit_libs_linux registered.
flutter: media_kit: WARNING: package:media_kit_native_event_loop not found.
flutter: Configure AudioServiceLinux.
method call InitSystemTray
SystemTray::set_system_tray_info title: , icon_path: /usr/share/spotube-bin/data/flutter_assets/assets/spotube-logo.png, toolTip: (null)
method call CreateContextMenu
value_to_menu_item type:label, label:Show/Hide
value_to_menu_item type:label, label:Play/Pause
value_to_menu_item type:label, label:Next
value_to_menu_item type:label, label:Previous
value_to_menu_item type:label, label:Quit
flutter: setQueue() has not been implemented.
flutter: Get org.mpris.MediaPlayer2.Player.Volume not implemented
flutter: Get org.mpris.MediaPlayer2.Player.Volume not implemented
flutter: GetPosition(): 0:00:00.000000
flutter: GetPosition(): 0:00:00.000000
method call SetContextMenu
flutter: Get org.mpris.MediaPlayer2.Player.Volume not implemented
flutter: Get org.mpris.MediaPlayer2.Player.Volume not implemented
flutter: GetPosition(): 0:00:00.000000
flutter: GetPosition(): 0:00:00.000000
```

I have checked to see whether I have the dependencies installed, also tried downloading the tar.gz straight from github and running that, just in case... but get the same result...

**Comment by Mikesco3** [comment]
I ended up filing this bug https://github.com/KRTirtho/spotube/issues/978
Because I'm not sure if it is the same issue as here and I only get it on version `3.4.0`

**Comment by KRTirtho** [comment]
Is it fixed in 3.8.0?

**Comment by KRTirtho** [comment]
Closing as stale
